Add allowUnlimitedContractSize, fork, and unlocked_accounts options support (#72)
OpenZeppelin/openzeppelin-test-environment: docs/modules/ROOT/pages/getting-started.adoc

    artifactsDir: 'build/contracts', // Directory where contract artifacts are stored
  },

  node: { // Options passed directly to Ganache client
    gasLimit: 8e6, // Maximum gas per block
    gasPrice: 20e9 // Sets the default gas price for transactions if not otherwise specified.
    fork: 'https://mainnet.infura.io/v3/{token}@{blocknumber}, // An url to Ethereum node to use as a source for a fork

>     unlocked_accounts: ['0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B'], // Array of addresses specifying which accounts should be unlocked.

Reviewer: I'd document these separately: `unlocked_ accounts` makes no sense without `fork`, and I'm not sure how the other interact (can you fork and change `gasLimit`?). 

In any case, `fork` is quite specialized and will only be used by advanced users, I'd rather not cause confusion by listing it along with everything else. See the [Advanced Options](https://docs.openzeppelin.com/test-environment/0.1/getting-started#advanced-options) section.

Author: Totally agree on Advanced Options.

  },
};
```

==== `fork`and `unlocked_accounts`

> Allows to fork a chain and unlock specified accounts on it. Useful for running integration tests with other protocols.

Reviewer: ```suggestion
These options allow Test Environment's local blockchain to _fork_ an existing one instead of starting from an empty state. By using them you can test how your code interacts with live third party protocols (like the MakerDAO system) without having to deploy them yourself!

In forked mode, you will also be able to send transactions from any of the `unlocked_accounts` (even if you don't know their private keys!).
```

```javascript
// test-environment.config.js

module.exports = {
  node: { // Options passed directly to Ganache client
    fork: 'https://mainnet.infura.io/v3/{token}@{blocknumber}, // An url to Ethereum node to use as a source for a fork
    unlocked_accounts: ['0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B'], // Array of addresses specifying which accounts should be unlocked.

>     allowUnlimitedContractSize: false // Allows unlimited contract sizes.

Reviewer: ```suggestion
```

  },
};
```

==== `allowUnlimitedContractSize`

> Allows unlimited contract sizes while debugging. By enabling this flag, the check within the EVM for contract size limit of 24KB (see EIP-170) is bypassed. Enabling this flag will cause ganache-cli to behave differently than production environments.

Reviewer: ```suggestion
Allows unlimited contract sizes. By enabling this flag, the check within the EVM for contract size limit of 24kB (see https://eips.ethereum.org/EIPS/eip-170[EIP-170]) is bypassed. Useful when testing unoptimized contracts that wouldn't be otherwise deployable.

WARNING: Enabling this flag causes Ganache to behave differently from production environments.
```

```javascript
// test-environment.config.js

module.exports = {
  node: { // Options passed directly to Ganache client

>      allowUnlimitedContractSize: false // Allows unlimited contract sizes.

Reviewer: ```suggestion
     allowUnlimitedContractSize: true, // Allows unlimited contract sizes.
```
